aces/Loris
# [EEG] Add view/edit functionality and permissions for EEG BIDS derivatives

1. Added a new permission for a user to view and/or edit annotations for specific entries in the electrophysiology browser module. 
2. Created functions in the backend of the electrophysiology browser module to get and display derivative data from the database and to write derivative data to the database. There is no view/edit/delete functionality in the front-end added by this PR.
3. Added the ability to download annotation files for a given session. Created a function that updates the annotation files to match changes in the database before they are downloaded by a user. This function can be called by a cronjob using the `tools/update_annotation_files.php` to update all the files periodically.

**Note:** The `tools/updateDerivativesTest` was added for testing purposes and should be removed before merging.  

#### Testing instructions

To test the SQL patch: 
1. Source the `2021-04-23-annotation_permissions.sql` patch or refresh the database with the edited SQL permissions schema and see that the correct entries are added to the database.

To test the download functionality:
1. Check that your database includes some derivative datasets and files. 
2. Navigate to the 
3. See that the button to download annotation files is visible and clickable
4. Check that clicking this button correctly downloads a .tgz file with the correct annotation files
5. Add new data to the `physiological_annotation_instance` and `physiological_annotation_parameter` tables for the respective files. 
6. Re-download the files and see that they are correctly updated with the new information.

Related to #7345
